docs/src/developing/clients/jsonrpc-api.md

```diff
@@ -3206,6 +3206,7 @@ Simulate sending a transaction
   - `sigVerify: <bool>` - if true the transaction signatures will be verified (default: false)
   - `commitment: <string>` - (optional) [Commitment](jsonrpc-api.md#configuring-state-commitment) level to simulate the transaction at (default: `"finalized"`).
   - `encoding: <string>` - (optional) Encoding used for the transaction data. Either `"base58"` (*slow*, **DEPRECATED**), or `"base64"`. (default: `"base58"`).
+  - `useMostRecentBlockhash: <bool>` - (optional) if true the transaction recent blockhash will be ignored and overridden with the most recent blockhash. (default: false)
```

Comment:
We probably should error if `sigVerify` is true when this flag is set

Reply:
I added the recent_blockhash override after sig verification to avoid conflicts there. Due to RPC consistency issues I think it's still reasonable for a client to want enable both options. I don't think it does any harm to allow both but it is a little odd so if you feel strongly enough we can prohibit it

Reply:
Yeah, I think we should prohibit it for now. It doesn't sit right to me that we first verify a signature, then promptly modify the message that was just verified even though it doesn't really matter in this case.

Reply:
Fixed and added tests